iModelCore/GeomLibs/PublicAPI/Geom/CurveDetails.h

```diff
@@ -515,7 +515,15 @@ double m_arcAngle;  // (if this is positive) turns larger than this become arcs.
 double m_chamferAngle; // (if this is positive) "outer chamfers" are created with this max angle.
 double m_offsetDistance;
 bool   m_forceClosure;
-bool   m_unusedBool[8];
+bool m_allowSharpestCorners; // Whether to remove the internal turn angle upper bound for sharp corner construction.

```

Comment:
You don't appear to initialize `m_allowSharpestCorners` in the constructor. Can you do that, or give it a default value here? Otherwise it will be random bits.

Reply:
Sure. I initialize it to false. Thanks for the comment.